subsys/bootloader/provision/provision_flash.c

```diff
@@ -10,15 +10,23 @@
 #include <generated_dts_board.h>
 #include <errno.h>
 #include "../bootloader.h"
+#include <nrf.h>
+#include <assert.h>
+
 typedef struct {
 	u32_t s0_address;
 	u32_t s1_address;
 	u32_t num_public_keys;
-	u8_t pkd[1];
+	u32_t pkd[1];
 } provision_flash_t;
 
+#ifdef CONFIG_SOC_NRF9160
+static const provision_flash_t *p_provision_data =
+	(provision_flash_t *)NRF_UICR_S->OTP;
```

Comment:
Could you instead change the dts to point to OTP?

Reply:
We currently don't have any entries in DTS for the bootloader partitions, we have a temporary hack in a header file which will be replaced when the partition manager gets in. I will add it to our temporary hack file which lists all the DTS symbols.